ios/PurchasesHybridCommon/PurchasesHybridCommon.xcodeproj/project.pbxproj

```diff
@@ -61,10 +61,11 @@
 		57FFDE3F286A777D0089A57B /* StoreKitIntegrationTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 57FFDE3E286A777D0089A57B /* StoreKitIntegrationTests.swift */; };
 		57FFDE41286A77E30089A57B /* CommonFunctionality+async.swift in Sources */ = {isa = PBXBuildFile; fileRef = 57FFDE40286A77E30089A57B /* CommonFunctionality+async.swift */; };
 		57FFDE46286B606A0089A57B /* XCTestCase+Expectations.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2DD3D1842810A81F00A19EC6 /* XCTestCase+Expectations.swift */; };
+		77D5E72A2C233E2C002ECC77 /* Constants.swift in Sources */ = {isa = PBXBuildFile; fileRef = 77D5E7292C233E2C002ECC77 /* Constants.swift */; };
 		AE0B58267EE48614BBF43A9E /* Pods_PurchasesHybridCommonUI.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D02631302E453BBC6C5F5471 /* Pods_PurchasesHybridCommonUI.framework */; };
 		B1C9A7EEA6961CC6DE22BCAD /* Pods_PurchasesHybridCommon_PurchasesHybridCommonTests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = E05FA8EACFF88F020CF8B9BB /* Pods_PurchasesHybridCommon_PurchasesHybridCommonTests.framework */; };
 		B3105D7A287E3CC200CA935B /* MockSandboxEnvironmentDetector.swift in Sources */ = {isa = PBXBuildFile; fileRef = B3105D79287E3CC200CA935B /* MockSandboxEnvironmentDetector.swift */; };
-		C5335FED2A63D9B5B628774F /* BuildFile in Frameworks */ = {isa = PBXBuildFile; };
+		C5335FED2A63D9B5B628774F /* (null) in Frameworks */ = {isa = PBXBuildFile; };
```

Comment:
Hmm does this mean it's missing a reference to this file? Seems like there might be a bad reference on the `Frameworks` folder but not sure...

Reply:
Hmm according to https://github.com/CocoaPods/Xcodeproj/issues/574, this is an orphaned file, since it has no `fileRef`. The other files do, if you look at the lines above this one. 

- If Xcode is the one writing the `project.pbxproj` file, it will write `(null) in Frameworks` for an orphaned file.
- If Xcodeproj is the one writing the `project.pbxproj` file, it will write `BuildFile in Frameworks` for an orphaned file.

So what I think happened here is that the `project.pbxproj` file was last written with Xcodeproj, and on my local machine it was last written with Xcode, causing the diff.

My understanding from that Xcodeproj issue thread is that this was already an orphaned file, and this entire line should just be removed. Let me know if you come to the same or different conclusion!

Reply:
Yup! I think this should be removed, but ideally this happens from the Xcode IDE, which should reference this as a not found file or something like that, since there might be another reference in this project file.

Reply:
As discussed on Slack, it seems that https://github.com/RevenueCat/purchases-hybrid-common/pull/672 changed the reference to `Pods_PurchasesHybridCommon.framework`, orphaning the previous reference. I have removed the orphaned references. It builds fine locally. Let's see what CI thinks.

Reply:
CI says :shipit: